TouchEffect.Droid/PlatformTouchEff.cs

```
@@ -97,13 +101,18 @@ protected override void OnDetached()
             catch (ObjectDisposedException)
             {
                 //suppress exception
+            }
+            catch(Exception)
```

remove this empty catch